lib/workers/repository/process/extract-update.ts

```diff
@@ -175,7 +175,10 @@ async function fetchVulnerabilities(
   if (config.osvVulnerabilityAlerts) {
     try {
       const vulnerabilities = await Vulnerabilities.create();
-      await vulnerabilities.fetchVulnerabilities(config, packageFiles);
+      await vulnerabilities.applyVulnerabilityPackageRules(
```

Comment:
> to be semantically precise, package rules aren't applied but compiled into a list here. Applying them happens later here and in related places

🤔  fair point. I think we should name it `appendVulnerabilityPackageRules`. WDYT @viceice 

> rename the existing (and then private) fetchVulnerabilities to getOsvVulnerabilities (or similar)

The aim of this PR to allow later usage of the Vulnerabilities outside of the singleton for example on the dashboard.   
I think `fetchVulnerabilities` is fitting here. 
`fetch` indicates we are getting something outside of the core code base and we are returning the new internal interface instead of an `Osv.Vulnerability`.

Reply:
i think `appendVulnerabilityPackageRules` is ok here 🤔